00-HTML-CSS-basics/The Basics of HTML/Questions.txt

```diff
@@ -1,5 +1,11 @@
+General:
+
+I accidentally pushed a commint to the master branch.
+Can I remove it somehow with rebase?
```

Comment:
Try to delete the master branch (locally) and then pull it again and checkout. I'll look into google to give you a guide to make this. Anyway if you do that so you'll lose all your changes there.